docs/content/docs/usage/intl.md

```diff
@@ -40,203 +40,166 @@ Now create differents `.yml` files in a `translations` folder.
 If you want handle two languages, you have to create two files,
 with language key as filename.
 
-`en.yml`:
+`src/translation/en.yml`:
 
 ```yml
-locale: "en"
+header:
+  title: "Phenomic"
+  posts: "Posts"
 
-messages:
+content:
   phenomic-is-awesome: "Phenomic is awesome !"
 ```
 
-`fr.yml`:
+`src/translation/fr.yml`:
 
 ```yml
-locale: "fr"
+header:
+  title: "Phenomic"
+  posts: "Articles"
 
-messages:
+content:
   phenomic-is-awesome: "Phenomic est génial !"
 ```
 
-From now you have 2 variations: with or without redux.
+## Initialize react-intl
 
-Without redux the thing will be pretty simple:
+We will need the `flat` lib to transform our translations files into flatten objects.
+The `flat` lib will take our nested translations objects and flatten then.
 
-We will assume all your urls will contain the locale key as the first part
+So instead of have:
 
-```
-http://domain.tld/en
-http://domain.tld/fr
+```javascript
+{
+  header: {
+    title: "Phenomic",
+    posts: "Posts",
+  }
+}
 ```
 
-You can read the locale like this in your appContainer using ``location`` from
-react-router context
+We will have:
 
-```js
-const locale = location.pathname.replace(/^\//, "").split("/")[0]
+```javascript
+{
+  "header.title": "Phenomic",
+  "header.posts": "Posts",
+}
 ```
 
-For you homepage, you can choose to rely on a default locale in your code.
-
-This should be enough in most cases.
-
-When you have the locale, you can rely on it in ``AppContainer`` to initialize
-[``IntlProvider``](https://github.com/yahoo/react-intl/wiki/Components#intlprovider)
-
-// @todo document more
-
-That said, you might want something more complex, you can use the following
-section.
-
----
-
-## Using Redux
-
-### Create redux action
+```console
+npm i --save flat
+```
 
-We will define our `redux` actions constants.
-Create an `actions.js` file under a `constants` folder.
+We will assume all your urls will contain the locale key as the first part
 
-```js
-export const SET_LOCALE = "SET_LOCALE"
+```
+http://domain.tld/en
+http://domain.tld/fr
 ```
 
-### Create intl actions
-
-We need to implement the `setLocale` `redux` action.
+If it's not the case, add the `route` metadata to your markdown files :
 
-We will need the `flat` lib to transform our translations files into flatten objects
+Add a metadata `locale` to your `.md` files
 
-```console
-npm i -S flat
+```markdown
+---
+title: Your title
+route: en/your-title
```

Comment:
I remove `locale`metadata and add explaination to use `route` metadata if routes are not good